octopart.gs

```diff
@@ -205,6 +205,31 @@ function Part(part) {
 
     return this._part.datasheets[0].url;
   }
+  
+ 
+   //This fuction return only url of large and medium sized pictures.
+    this.getImageSets = function() {
```

Comment:
@shyne99, looks good. I would change the indentation in this function as it appears to be wrong and combine the two for loops into one with the two if's inside it.

I don't think we should limit sources even for "good" reasons. There are just too many sources to curate and decide what are worthwhile.